public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -677,3 +675,83 @@ export const parseHits = (payloadData: string): ParsedHit[] => {
     return [];
   }
 };
+
+export const InjectElementsIntoGrid = (
```

Comment:
the helper_functions file is getting too lang, it might be better to split this into a different file especially it's a custom react hook

also it's better to name hooks with `use` prefix to help linters
```suggestion
export const useInjectElementsIntoGrid = (
```

and could you add some comments on top or make naming more specific, for example what elements are injected

Reply:
Made new file public/components/trace_analytics/components/common/shared_components/component_helper_functions.tsx
And moved useInjectElementsIntoGrid = ( to it.
Added comment above
// Injects the size warning and Load buttons into the corners of EUI Data Grid